From: Michal Hocko <mhocko@suse.cz>
Subject: mm: inform when the memory mirroring is enabled
Patch-mainline: never, suse internal
References: bnc#987598, fate#319143

For a better supportability of this feature we should be informed when it is enabled.
This patch doesn't guarantee that the mirrored memory is really used in the end but
we have other patch which warns when a movable vs. kernel memory ratio is too high
so this is not strictly needed.

Signed-off-by: Michal Hocko <mhocko@suse.cz>

---
 mm/page_alloc.c |    1 +
 1 file changed, 1 insertion(+)

--- a/mm/page_alloc.c
+++ b/mm/page_alloc.c
@@ -6287,6 +6287,7 @@ static int __init cmdline_parse_kernelco
 	/* parse kernelcore=mirror */
 	if (parse_option_str(p, "mirror")) {
 		mirrored_kernelcore = true;
+		pr_info("Kernel memory mirroring enabled.\n");
 		return 0;
 	}
 
